week6/ks21_trash_bins.py

```diff
@@ -0,0 +1,36 @@
+def parse_input():
```

Comment:
An intuitive solution, but this approach will potentially visit same cell too much times when you have 1 at the leftmost and the rightmost and you have million houses. You can avoid that by just iterating all the cells once.